cards/document-vertical-standard/component.js

```diff
@@ -0,0 +1,55 @@
+{{> cards/card_component componentName='document-vertical-standard' }}
+
+class document_vertical_standardCardComponent extends BaseCard['document-vertical-standard'] {
+    constructor(config = {}, systemConfig = {}) {
+        super(config, systemConfig);
+    }
+
+    /**
+     * This returns an object that will be called `card`
+     * in the template. Put all mapping logic here.
+     *
+     * @param profile profile of the entity in the card
+     */
+    dataForRender(profile) {
+        const linkTarget = AnswersExperience.runtimeConfig.get('linkTarget') || '_top';
+
+        return {
+            title: profile.name, // The header text of the card
+            // subtitle: '', // The sub-header text of the card
+            url: profile.landingPageUrl, // If the card title is a clickable link, set URL here
```

Comment:
Would these ever be populated? Plus is there a field like "c_department" always present on a profile? If they're not present what happens?

Reply:
subtitle was a miss. landingPageURL is a universal field, so I left it in since I could see clients wanting to use it. linkTarget is configurable and all cards reference it, and should be populated. That said, if we don't want any url fields maybe linkTarget isn't getting us much done here.
If a value isn't present then in the handlebars file we don't render it.
I'm going to remove subtitle now (as a populated field, but keep it commented out), but let me know about the other two.